app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BridgeResponse.java

```diff
@@ -0,0 +1,32 @@
+package com.eveningoutpost.dexdrip.UtilityModels;
+
+import java.nio.ByteBuffer;
+import java.util.LinkedList;
+
+import lombok.Data;
+
+/**
+ * Created by jamorham on 16/03/2018.
+ */
+
+@Data
+public class BridgeResponse {
+
+    public final LinkedList<ByteBuffer> send;
```

Comment:
Please make the send private and manipulate it through setters/getters.
Otherwise it is hard to say when this class is changing.

Reply:
The reason it is done like this is so you can do a cleaner syntax of:

` reply.send.add(allowNewSensor);`

instead of having to do:

`reply.getSend().add(allowNewSensor)`

I think it is easier to read like that myself.

Reply:
I think that the best way to do it is to add a function add on the BridgeResponse 
in that case, adding data to it will only be done inside it's code. Otherwise it is not possible to know  who is manipulating it's data.

Reply:
Yes this obeys object orientated principles. noted.